exaudfclient/base/exaudflib/BUILD

```diff
@@ -46,7 +46,8 @@ cc_library(
 cc_library(
     name = "header",
     hdrs = [
-        "exaudflib.h",
+        "exaudflib.h", "vm/swig_vm.h", "vm/java_vm.h", "vm/python_vm.h",
+        "swig/swig_common.h", "swig/swig_meta_data.h", "swig/swig_result_handler.h", "swig/swig_table_iterator.h"
```

Comment:
Maybe we could still split up this in further packages....

Reply:
you mean, how you did it here, into one file per metadata, result handler and table iterator? if yes, that is fine.

Reply:
Actually I meant to create a  new cc_library:
```swig_header```
and then include in the header cc_library. But I think it would increase complexity in an unnecessary way.